x-pack/plugins/observability_solution/dataset_quality/common/types/common.ts

```diff
@@ -13,3 +16,11 @@ export interface Coordinate {
   x: number;
   y: Maybe<number>;
 }
+
+export interface BasicDataStreamStats {
```

Comment:
[nit] I'd call this one `DataStreamDetailsStats` or something similar just to point out that we would using that one within `dataStreamDetails` (aka flyout, later page) instead of basic one.

Reply:
So i had 2 things in my mind when i chose this name
1. We already have `dataStreamStats` and `dataStreamDetails` and i actually though to name it how you suggested but that would again be one more variable looking, sounding similar.
2. Why i called it Basic ? This interface is used for `useRedirectLink` function. This locator function is generic to the whole Dataset Quality Page. Also this locator needs bare minimum explicit parameters to work like the ones defined here. If i call it DataStreamDetailsStats, that would confuse that this object may have more information about the DataStream where as in reality we don't need all those information. We just need these 5 properties.

Reply:
What about just `DataStream`? looking it closely this object doesn't contain any stats

Reply:
We settled offline on `BasicDataStream`